src/components/Button/Button.stories.tsx

```diff
@@ -22,16 +23,22 @@ Default.args = { children: 'Button' };
 
 export const DefaultWithIconBefore = Template.bind({});
 DefaultWithIconBefore.args = {
-  children: 'Button',
-  iconPosition: 'before',
-  iconName: 'chevron-left',
+  children: (
+    <>
+      <Icon aria-hidden="true" focusable={false} name="chevron-left" />
```

Comment:
Side note: when would we want the icon to be focusable anyway? Why not put it in a `Button`? We recently found a bug where tooltips aren't read out in NVDA unless the trigger is in a `Button`, so I don't think tooltips is the intended use case here.

Reply:
I had the same Q while merging in SvgIcon props & decided to just remove `focusable` 😬  I think it should default to `false` tho so you shouldn't need to pass it here. also secretly the component passes `aria-hidden={!title}` so you shouldn't need to explicitly set aria-hidden to true either

Reply:
Great, thanks! I'll wait until your PR merges so I can also add the `purpose` prop to these new icons. I'll also clean up the `focusable`, `aria-hidden`, and `aria-label` vs `title` stuff.

Reply:
just merged, fyi!

Reply:
Thanks!